tools/squidpy/main_macros.xml

```diff
@@ -1,7 +1,7 @@
 <macros>
-    <token name="@TOOL_VERSION@">1.1.2</token>
+    <token name="@TOOL_VERSION@">1.4.1</token>
     <token name="@PROFILE@">20.01</token>
-    <token name="@VERSION_SUFFIX@">2</token> 
+    <token name="@VERSION_SUFFIX@">3</token> 
```

Comment:
this can be reset to 0 if you bump the major version

Reply:
sounds good, thanks for looking this over! this is a minor revision bump of squidpy, (`1.1.2` -> `1.4.1`), is this what you meant? Should I still change the VERSION_SUFFIX to `0`, or did you mean change the VERSION_SUFFIX to `1.0`?